docs/spec/openapi/public-docs.yaml

```diff
@@ -0,0 +1,5703 @@
+                - $ref: '#/components/schemas/SpanEvent'
+                - $ref: '#/components/schemas/TransactionEvent'
+      responses:
+        '202':
+          description: Successful response; all events succeeded.
+  /intake/v3/rum/events:
+    post:
+      summary: Send RUM events
+      description: >
+        NOTE: Most users do not need to interact directly with the events intake
+        API.
+      operationId: postRumEventIntakeV3
+      tags:
+        - event intake
+      requestBody:
+        content:
+          application/ndjson:
+            schema:
+              anyOf:
+                - $ref: '#/components/schemas/ErrorEventv3'
+                - $ref: '#/components/schemas/MetadataEventv3'
+                - $ref: '#/components/schemas/SpanEventv3'
+                - $ref: '#/components/schemas/TransactionEventv3'
+      responses:
+        '202':
+          description: Successful response; all events succeeded.
+  /opentelemetry.proto.collector.metrics.v1.MetricsService/Export:
+    post:
+      summary: Send OTLP metrics
+      description: >
+        APM Server supports receiving metrics over the OpenTelemetry Protocol
+        (OTLP). This API uses the OTLP/gRPC communication protocol.
+      operationId: postOtlpGrpcMetrics
+      tags:
+        - opentelemetry intake
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        '200':
+          description: OK
+  /opentelemetry.proto.collector.trace.v1.TraceService/Export:
+    post:
+      summary: Send OTLP traces
+      description: >
+        APM Server supports receiving traces over the OpenTelemetry Protocol
+        (OTLP). This API uses the OTLP/gRPC communication protocol.
+      operationId: postOtlpGrpcTraces
+      tags:
+        - opentelemetry intake
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        '200':
+          description: OK
+  /opentelemetry.proto.collector.logs.v1.LogsService/Export:
+    post:
+      summary: Send OTLP logs
+      description: >
+        APM Server supports receiving logs over the OpenTelemetry Protocol
+        (OTLP). This API uses the OTLP/gRPC communication protocol.
+      operationId: postOtlpGrpcLogs
+      tags:
+        - opentelemetry intake
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        '200':
+          description: OK
+  /v1/metrics:
+    post:
+      summary: Send OTLP metrics
+      description: >
+        APM Server supports receiving metrics over the OpenTelemetry Protocol
+        (OTLP). This API uses the OTLP/HTTP (protobuf) communication protocol.
+      operationId: postOtlpHttpMetrics
+      tags:
+        - opentelemetry intake
+      requestBody:
+        content:
+          application/json:
```

Comment:
otlp/http should use content-type `application/x-protobuf` https://opentelemetry.io/docs/specs/otlp/#binary-protobuf-encoding. But not sure if it is supported in openapi